experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { EventEmitter } from "events";
+import { IAudience } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import {
+    LoadableObjectClass,
+} from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+export class FluidContainer
+    extends EventEmitter
+    implements
+        Pick<Container, "audience" | "clientId" | "close" | "closed" | "connected" >,
+        Pick<RootDataObject, "initialObjects"> {
+    private readonly _container: Container | undefined = undefined;
+    private readonly _rootDataObject: RootDataObject | undefined = undefined;
+
+    public constructor(container: Container, dataObject: RootDataObject) {
+            super();
+            this._container = container;
+            this._rootDataObject = dataObject;
+
+            container.on("connected", (id: string) =>  this.emit("connected", id));
+    }
+
+    static async create(container: Container) {
+        const rootDataObject = (await container.request({ url: "/" })).value;
+        return new FluidContainer(container, rootDataObject);
+    }
+
+    private get container(): Container {
+        assert(this._container !== undefined, "container undefined");
+        return this._container;
+    }
+
+    private get rootDataObject(): RootDataObject {
+        assert(this._rootDataObject !== undefined, "defaultObject undefined");
+        return this._rootDataObject;
+    }
```

Comment:
You don't need either of these since you are setting both of them in the constructor. The _ types that you have don't need to be defined as undefined because they can never be undefined.